USAGE.md

```diff
@@ -74,16 +61,44 @@ The GDL performs many checks on datasets. If any of the checks fail the dataset
 - Every metadata and asset URL must be readable by the GDL.
 - A dataset *may* refer to the same asset more than once. All references to the same asset must have the same multihash. That is, having a SHA-1 and a SHA-256 checksum for the same file will be considered invalid, even if both checksums are valid. This is to enable a simpler checksum validation.
 
-## Data Lake Endpoints Usage
+
+# Authentication and authorization
+Data Lake allows read/write access for users authorized by SAML identity provider
+(`DATALAKE_SAML_IDENTITY_PROVIDER_ARN`) configured during deployment time (see
+[README](README.md#aws-infrastructure-deployment-cdk-stack)).
+
+Example of AWS service account authentication and authorization in to Data Lake users role via
+Azure:
+
+- `~/.aws/config` file
+    ```
+    [profile data-lake-users]
+    azure_tenant_id=<AZURE_TENANT_ID>
+    azure_app_id_uri=https://signin.aws.amazon.com/saml\#<AZURE_APP_ID>
+    azure_default_username=<AZURE_USER_NAME>
+    azure_default_role_arn=arn:aws:iam::<DATALAKE_AWS_ACCOUNT_ID>:role/<DATALAKE_USER_ROLE_NAME>
+    azure_default_duration_hours=12
+    azure_default_remember_me=true
+    ```
+
+- Log in to Data Lake AWS account using Data Lake users role
+```bash
+aws-azure-login --profile data-lake-users
+```
+
+
+# Data Lake endpoints
 
 There are two end user interaction points in GDL:
 
 - A [dataset space endpoint](TODO), to create, get, update or delete individual datasets, and to list all datasets
 - A [dataset versions endpoint](TODO), to create new versions of datasets. The S3 files which constitute the dataset are all linked to a specific dataset version.
 
-Both of these are implemented as AWS Lambda functions, which means they can be run ("invoked") either via the AWS web interface (links above) or via any tool using the AWS API, such as the commands below.
+Both of these are implemented as AWS Lambda functions, which means they can be run ("invoked")
```

Comment:
Can you share your Markdown line split configuration or disable it? We keep writing long lines and then splitting them, which makes the diffs unnecessarily big.

Reply:
I don't think keeping those long lines is a good idea. I would make equally short as our source code is.

Reply:
I agree to have something formatting lines. For now I keep it as is and we should try to fix it. I was trying to use prettier, but looks like it needs some configuration.